unitTests/CMakeLists.txt

```diff
@@ -87,6 +87,11 @@ blt_add_executable( NAME testFloatingPointExceptions
                     OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
                     DEPENDS_ON gtest lvarray ${lvarray_dependencies} )
 
+# Need to avoid optimization to catch invalid operations
+if( APPLE AND ${CMAKE_CXX_COMPILER} MATCHES "clang" )
```

Comment:
Ahh is this because APPLE + "clang" doesn't respect the no-inline attribute?

Reply:
I think so ... I am not a Mac expect!